client/index.js

```diff
@@ -12,6 +12,10 @@ import { PageLayout, EmbedLayout, PrimaryLayout as NoticeArea } from './layout';
 import 'wc-api/wp-data-store';
 import { withSettingsHydration } from '@woocommerce/data';
 
+// Modify webpack pubilcPath at runtime based on location of WordPress Plugin.
+// eslint-disable-next-line no-undef,camelcase
+__webpack_public_path__ = global.wcAdminAssets.path;
```

Comment:
was always wondering how to solve this issue with dynamic loading, awesome!

Reply:
Looks like this is causing a console error on non-WC-Admin pages. I'm investigating..

Reply:
Red herring! This isn't causing any issue.